client/hooks/useUserCrudOperations.ts

```diff
@@ -27,9 +27,9 @@ const postNewUserProfile = async (): Promise<User> => {
   }
 };
 
-const putExistingUserProfile = async (): Promise<User> => {
+const putExistingUserProfile = async (data: Partial<User>): Promise<User> => {
   try {
-    const response = await axiosClient.put('/v1/users');
+    const response = await axiosClient.put('/v1/users', data);
```

Comment:
this is probably perfectly OK, but could use PATCH here since we're only updating 1 field on the user, no need to update the whole user record